sentry-android-core/src/test/java/io/sentry/android/core/AndroidTransportGateTest.kt

```diff
@@ -8,20 +8,38 @@ import kotlin.test.BeforeTest
 import kotlin.test.Test
 import kotlin.test.assertNotNull
 import org.junit.runner.RunWith
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
 
 @RunWith(AndroidJUnit4::class)
 class AndroidTransportGateTest {
 
     private lateinit var context: Context
+    private lateinit var transportGate: AndroidTransportGate
 
     @BeforeTest
     fun `set up`() {
         context = ApplicationProvider.getApplicationContext()
+        transportGate = AndroidTransportGate(context, mock())
     }
 
     @Test
-    fun `isConnected won't throw exception`() {
-        val transportGate = AndroidTransportGate(context, mock())
+    fun `isSendingAllowed won't throw exception`() {
```

Comment:
Maybe instead of `does not throw ` it could be `is not null`.
Since we can assert something here.

Reply:
good catch